contrib/linux/actions/rsync.yaml

```diff
@@ -9,21 +9,20 @@
             type: 'string'
             description: 'List of files/directories to to be copied'
             required: true
-            position: 1
+            position: 0
```

Comment:
The intent is that the parameters are passed to the `cmd` parameter which works as a template for the positions of the parameters.  I rarely need this anymore but this was a common pattern early on for giving user friendly parameters in the meta and then building the proper linux command in the `cmd` default.